### PR TITLE
updated avatar component name, added ability to change username (only works server side for now), updated signin page, updated Footer

### PR DESCRIPTION
**Pull request by rsandb**
A bunch of these commits were already added so I'm not sure why they're showing up here. The only one that I'm requesting is "[de86903](https://github.com/LAION-AI/Open-Assistant/pull/91/commits/de869033bf9edf3c051d01c264637b3d286a65ae)"

### CODE REVIEW

File: website/src/pages/account/index.tsx
Comment (fozziethebeat) [inline website/src/pages/account/index.tsx:7]: Can you use absolute imports for these? (`src/components/Footer`)

File: website/src/pages/account/edit.tsx
Comment (fozziethebeat) [inline website/src/pages/account/edit.tsx:8]: Can you use absolute imports here?

File: website/src/pages/account/edit.tsx
Comment (fozziethebeat) [inline website/src/pages/account/edit.tsx:7]: orderwise I like to treat `@` imports as first, can you put this first?

File: website/src/pages/account/edit.tsx
Comment (fozziethebeat) [inline website/src/pages/account/edit.tsx:20]: Just to ask, why is the url `../api/username`? Shouldn't `/api/username` work?

**Review by fozziethebeat — CHANGES_REQUESTED** [review]

**Comment by fozziethebeat** [comment]
Will approve after I see the conflicts are merged in and the small style changes are made.

File: website/src/pages/account/edit.tsx
Comment (rsandb) [reply website/src/pages/account/edit.tsx]: Yeah it does, didn't catch that thanks!

File: website/src/middleware.ts
Comment (AbdBarho) [inline website/src/middleware.ts:7]: /grading
and
/summarize
are deprecated, you can remove them.

File: website/src/pages/404.tsx
Comment (AbdBarho) [inline website/src/pages/404.tsx:14]: could you please also remove the header and footer here?

File: website/src/pages/_app.tsx
Comment (AbdBarho) [inline website/src/pages/_app.tsx:43]: why `pageProps.session`? is `session` not enough?

File: website/src/pages/account/edit.tsx
Comment (AbdBarho) [inline website/src/pages/account/edit.tsx:9]: could you remove unused imports?

File: website/src/pages/account/index.tsx
Comment (AbdBarho) [inline website/src/pages/account/index.tsx:8]: could you remove unused imports?

File: website/src/pages/api/username.tsx
Comment (AbdBarho) [inline website/src/pages/api/username.tsx:15]: is it ok if any of `session?.user?.email` is undefined?

If not, remove all question marks.

**Review by AbdBarho — COMMENTED** [review]
Thanks!

**Review by AbdBarho — APPROVED** [review]
Thanks!

LGTM

We will have to wait for @fozziethebeat to approve.

**Review by k-nearest-neighbor — APPROVED** [review]
Looks great!

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**
